operators/hpe-csi-operator/deploy/olm-catalog/hpe-csi-driver-operator/2.0.0/hpe-csi-driver-operator.v2.0.0.clusterserviceversion.yaml

```diff
@@ -147,11 +146,6 @@ spec:
             path: secret.password
             x-descriptors:
               - 'urn:alm:descriptor:com.tectonic.ui:password'
-          - displayName: CSP service hostname
-            description: CSP service hostname or IP address
-            path: secret.serviceName
-            x-descriptors:
-              - 'urn:alm:descriptor:com.tectonic.ui:text'
```

Comment:
Why is this being removed?

Reply:
serviceName is handled internally by the code templates